src/ServiceControlInstaller.Engine/Accounts/UserAccount.cs

```diff
@@ -26,6 +26,11 @@ public bool IsLocalService()
             return SID.IsWellKnown(WellKnownSidType.LocalServiceSid);
         }
 
+        bool IsServiceAccount()
+        {
+            return Domain == "NT SERVICE";
```

Comment:
The `NT SERVICE` check in the `LocalizedNTAuthority` method uses `Domain.Equals("NT SERVICE", StringComparison.CurrentCulture)`  Does it matter that this is different? Could there be languages where it is not the same spelling?